### PR TITLE
[DF] Low Priority Fixes

### LINKED ISSUE #211: [df] ambassadors, council & notifications
### Hugo

- [x] * On decline / accept / claim buttons, call endpoints again and not refresh page
- [x] * As council member I have “my communities” tab. Unless it's ambassador, it's not supposed.
- [x] * As councilmember, I have a notification that has no title. It only shows “2 days ago”

### Bernardo

- [x] * As ambassador, I don’t see the “requests” option on the sidebar
- [x] * When ambassador claims community, update only buttons, not the whole page
- [x] * If council member has other roles beyond being council member, doesn’t see the proposals menu option
- [x] * If the beneficiary claims the UBI and the  community then has no funds, it shows “no funds”, instead of the countdown. Should shown the countdown regardless

### PR DESCRIPTION
**Pull request by hugomarquesdev**
#211

### CODE REVIEW

File: src/views/Notifications.tsx [hunk 1]
```diff
@@ -110,6 +110,14 @@ const Notifications: React.FC<{ isLoading?: boolean }> = (props) => {
...
                 break;
             } 
+            case 2: { 
+                router.push(`/communities/${contentId}`);
+                break;
+            } 
+            case 3: { 
+                router.push(`/communities/${contentId}`);
+                break;
+            }
```
Comment (obernardovieira) [inline src/views/Notifications.tsx:120]: https://stackoverflow.com/a/13207939 ☺️

Comment (hugomarquesdev) [reply src/views/Notifications.tsx]: thanks for the tip! 😄

**Review by obernardovieira — CHANGES_REQUESTED** [review]
* If the ambassador is also a manager, it doesn't see the ambassador menu options such as community requests. Test account Ambassador 1 has that issue.
* Beneficiary sees both countdown and no funds warning. The no funds warning should be at the top of the page if the countdown is active. See design.
* Test account Council member 2, that is also ambassador and manager, doesn't see council member pages on sidebar, neither ambassador, only manager

**Comment by hugomarquesdev** [comment]
> * If the ambassador is also a manager, it doesn't see the ambassador menu options such as community requests. Test account Ambassador 1 has that issue.
> * Beneficiary sees both countdown and no funds warning. The no funds warning should be at the top of the page if the countdown is active. See design.
> * Test account Council member 2, that is also ambassador and manager, doesn't see council member pages on sidebar, neither ambassador, only manager

@obernardovieira 
- the sidebar bugs are fixed on the high priority fixes PR that isn't yet merged.
- i'll fix the countdown + no funds, didn't check design, sorry 😅

**Review by obernardovieira — CHANGES_REQUESTED** [review]
I did a claim with "beneficiary 4" https://alfajores-blockscout.celo-testnet.org/tx/0xea16720c247f1a73f9f5e8d352d7ac04847d245e75acdca76d5bbb4ce9ce84ff/token-transfers. As beneficiary I see the community has no funds, but i don't see any countdown.

**Comment by hugomarquesdev** [comment]
> I did a claim with "beneficiary 4" https://alfajores-blockscout.celo-testnet.org/tx/0xea16720c247f1a73f9f5e8d352d7ac04847d245e75acdca76d5bbb4ce9ce84ff/token-transfers. As beneficiary I see the community has no funds, but i don't see any countdown.

@obernardovieira 
from the devs call we've made I understood that if the community has no funds it doesn't know when it will have funds again. it can be days, weeks, etc... when that happens, the user has no countdown. 

basically what I understood was:
`hasFunds` and `isClaimable`: show claim btn
`hasFunds` and `isClaimable: false`: show countdown
`hasFunds: false`: doesn't show countdown (whether `isClaimable` is true or not) because even if `isClaimable` is true there are no funds, so, the beneficiary cant claim even when the countdown ends

from what you've commented, you want to show the countdown when `hasfunds: false `and `isClaimable: true` (the community has no funds)?

sorry for the long message and for my misconception once again 😣

File: src/components/UserMenu.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /* eslint-disable react-hooks/rules-of-hooks */
 import { usePrismicData } from '../libs/Prismic/components/PrismicDataProvider';
+import _ from 'lodash';
```
Comment (psoaresbj) [inline src/components/UserMenu.tsx:3]:
```suggestion
import kebabCase from 'lodash/kebabCase';
```

File: src/components/UserMenu.tsx [hunk 3]
```diff
@@ -11,8 +12,11 @@ export const getUserMenu = (roles: string[]) => {
         const activeRoles: any[] = []
 
         userConfig?.map((prismicData: { uid: string; data: { asideMenuItems: { items: any; }[]; }; }) => 
-            rolesArr?.map(role => 
-                prismicData?.uid === role && activeRoles?.push(prismicData?.data?.asideMenuItems[0]?.items)
+            rolesArr?.map(role => {
+                    const roleKebabCase = _.kebabCase(role);
```
Comment (psoaresbj) [inline src/components/UserMenu.tsx:16]:
```suggestion
                    const roleKebabCase = kebabCase(role);
```

**Review by psoaresbj — APPROVED** [review]

**Pull request merged**